assets/static/images/Facebook.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why are we using custom icons instead of Fontawesome? https://fontawesome.com/icons/facebook-square
Was this a requirement?

Reply:
These are the icons that were uploaded by Harry to Invision